src/main/java/persistence/sql/dml/PersonRowMapper.java

```diff
@@ -0,0 +1,36 @@
+package persistence.sql.dml;
+
+import jdbc.RowMapper;
+
+import java.io.File;
+import java.lang.reflect.Field;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class PersonRowMapper implements RowMapper<Person> {
+
+    @Override
+    public Person mapRow(ResultSet rs) {
+        Person person = new Person();
+        try {
+            Field idField = person.getClass().getDeclaredField("id");
+            idField.setAccessible(true);
+            idField.set(person, rs.getLong("id"));
+
+            Field nameField = person.getClass().getDeclaredField("nick_name");
+            nameField.setAccessible(true);
+            nameField.set(person, rs.getString("nick_name"));
+
+            Field ageField = person.getClass().getDeclaredField("old");
+            ageField.setAccessible(true);
+            ageField.set(person, rs.getInt("old"));
+
+            Field emailField = person.getClass().getDeclaredField("email");
+            emailField.setAccessible(true);
+            emailField.set(person, rs.getString("email"));
+
+        } catch (NoSuchFieldException | IllegalAccessException | SQLException e) {
+        }
+        return person;
+    }
```

Comment:
현재 코드에서는 위 로직에서 예외가 발생하고 있음에도 예외를 catch 하여 어떠한 행위도 하고 있지 않기 때문에 에러를 파악하기 어렵습니다! 예외를 catch 하실 때는 꼭 log를 남겨주시거나 exception을 발생시켜주시면 좋을 것 같아요!

```suggestion
    @Override
    public Person mapRow(ResultSet rs) {
        Person person = new Person();
        try {
            Field idField = person.getClass().getDeclaredField("id");
            idField.setAccessible(true);
            idField.set(person, rs.getLong("id"));

            Field nameField = person.getClass().getDeclaredField("nick_name");
            nameField.setAccessible(true);
            nameField.set(person, rs.getString("nick_name"));

            Field ageField = person.getClass().getDeclaredField("old");
            ageField.setAccessible(true);
            ageField.set(person, rs.getInt("old"));

            Field emailField = person.getClass().getDeclaredField("email");
            emailField.setAccessible(true);
            emailField.set(person, rs.getString("email"));

        } catch (NoSuchFieldException | IllegalAccessException | SQLException e) {
            throw new RuntimeException(e);
        }
        return person;
    }
```